test/parallel/test-async-wrap-getasyncid.js

```diff
@@ -66,6 +66,12 @@ function testInitialized(req, ctor_name) {
 }
 
 
+{
+  // TODO: determine the rigth way to expose promise wrap.
+  new Promise((res) => res(5));
```

Comment:
So does this mean that we do not need to expose a "getAsyncId" operation for promises (adding it will have performance implications so it would be great if it isn't needed)?